powerhome/playbook
# [PLAY-589] ♻️ Typescript Conversion: Title Count

#### Screens

<img width="573" alt="Screen Shot 2023-02-13 at 1 13 14 PM" src="https://user-images.githubusercontent.com/73671109/218539058-c7c9f822-506a-4bd9-bf41-925dbe89dd94.png">

#### Breaking Changes

NO

#### Runway Ticket URL

[Runway Ticket](https://nitro.powerhrg.com/runway/backlog_items/PLAY-589)

#### How to test this

Use the title count kit and ensure it works as expected after typescript conversion

#### Checklist:
